task/01-strings-tasks.js

```diff
@@ -221,9 +245,20 @@ function getRectangleString(width, height) {
  *
  */
 function encodeToRot13(str) {
-    throw new Error('Not implemented');
+	const input = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+	const output = 'NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm';
+	let result = '';
+
+	str.split('').forEach((element) => {
```

Comment:
use https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/map 
instead of foreach